### PR TITLE
Use default `Neff` in test_nonlin_camb_power

### LINKED ISSUE #964: Update `Neff = 3.044` (and revert #1024)
https://arxiv.org/abs/2008.01074
https://arxiv.org/abs/2012.02726

### PR DESCRIPTION
**Pull request by hsinfan1996**
close #964 and revert #1024

### CODE REVIEW

**Comment by hsinfan1996** [comment]
With conda-installed `isitgr`, we will not be able to install the latest camb from conda. The solution is to move one or both of them to pip. Is this an acceptable solution?

File: .github/environment.yml [hunk 1]
```diff
@@ -8,12 +8,13 @@ dependencies:
   - pyyaml
   - numpy
   - scipy
-  - camb=1.3.6
-  - isitgr
+  - camb
+  #- isitgr
```
Comment (damonge) [inline .github/environment.yml:12]: Why did you leave this commented out?

**Review by damonge — COMMENTED** [review]
Just one thing

**Comment by damonge** [comment]
Fine to install isitgr from pip. Whatever works.

File: .github/environment.yml (hunk 1 above)
Comment (hsinfan1996) [reply .github/environment.yml:12]: I will remove it. I will also need to remove the -v arg in pip install before finalizing this PR.

Comment (damonge) [reply .github/environment.yml:12]: OK great. I was not opposed to leaving the -v in, so up to you. I think we can merge after that, so I'll approve now so you don't have to wait for me.

**Review by damonge — APPROVED** [review]

**Pull request merged**